remoteappmanager/db/orm.py

```diff
@@ -11,28 +11,35 @@
 Base = declarative_base()
 
 
+class IdMixin(object):
+    """Base class to provide an id"""
+    id = Column(Integer, primary_key=True)
+
+    @classmethod
+    def from_id(cls, session, id):
+        return session.query(cls).filter(cls.id == id).one()
+
+
 class UserTeam(Base):
     """ The user (n <-> n) team association table """
     __tablename__ = "user_team"
     user_id = Column(Integer, ForeignKey('user.id'), primary_key=True)
     team_id = Column(Integer, ForeignKey('team.id'), primary_key=True)
 
 
-class User(Base):
+class User(IdMixin, Base):
```

Comment:
Why not have a new base class subclass of `Base` for all those table entries that have the `id` attribute. Is the use of mixins idiomatic to how orms are setup?